test/inject/test_find_content.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import annotations
+
+from pathlib import Path
+from unittest import TestCase
+
+from pytest import raises
+
+from flash_patcher.exception.error_manager import ErrorManager
+from flash_patcher.exception.injection import InjectionError
+from flash_patcher.inject.find_content import FindContentManager
+from flash_patcher.util.file_io import read_safe
+
+# pylint: disable=wrong-import-order
+from test.test_util.get_patch_context import get_add_patch_context, get_replace_patch_context
+
+class FindContentManagerSpec (TestCase):
+
```

Comment:
```suggestion
"""Some docstring."""
```
Unnecessary whitespace/missing docstring.

Reply:
This whitespace seems to be standard in our code. We should either keep it here or change it everywhere (any thoughts on which approach we should take?).

Also, all of our test case classes don't have a docstring, as the name should always follow the convention of `[base class name]Spec`, where `[base class name]` is the class whose behavior we are testing.

Reply:
Change everywhere, I'd say.

And, alright, that makes sense.